assets/js/modules/analytics-4/datastore/audiences.js

```diff
@@ -637,15 +637,19 @@ const baseActions = {
 			return { error };
 		}
 
-		// Expire new badges for initially configured audiences.
-		yield commonActions.await(
-			dispatch( CORE_USER ).setExpirableItemTimers(
-				configuredAudiences.map( ( slug ) => ( {
-					slug: `${ AUDIENCE_ITEM_NEW_BADGE_SLUG_PREFIX }${ slug }`,
-					expiresInSeconds: 1,
-				} ) )
-			)
-		);
+		if ( configuredAudiences.length > 0 ) {
```

Comment:
This fixes a minor bug, calling `setExpirableItemTimers()` with an empty list results in an error being returned and logged to the console.

Reply:
Nice one. Thanks for fixing it, @techanvil 👍